libs/blocks/global-navigation/features/profile/dropdown.js

```diff
@@ -157,7 +157,7 @@ class ProfileDropdown {
   decorateSignOut() {
     const signOutLink = toFragment`
       <li>
-        <a href="#" class="feds-profile-action" daa-ll="${this.placeholders.signOut}">${this.placeholders.signOut}</a>
+        <button class="feds-profile-action" daa-ll="${this.placeholders.signOut}">${this.placeholders.signOut}</button>
```

Comment:
The computed font is now Arial for me, we should use Adobe's font family.
<img width="623" alt="Screenshot 2023-08-31 at 16 32 04" src="https://github.com/adobecom/milo/assets/408175/8d30d969-f517-4686-b590-c9c5440fb23e">

Reply:
Other things to look out for after you change the `font-family`: `font-size` and `line-height`; you need to set them explicitly otherwise browser defaults will be applied.